src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+import { getUserById } from '../utils/GetUserByld';
+import { User } from '../types/User';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user: User = getUserById(todo.userId);
```

Comment:
Ensure that the `getUserById` function handles cases where a user might not be found. Consider adding a fallback or error handling mechanism if `getUserById` returns `undefined`.